package.json

```diff
@@ -1,49 +1,240 @@
 {
-    "name": "yarn-spinner",
-    "displayName": "Yarn Spinner",
-    "description": "Adds support for the Yarn Spinner dialogue language.",
-    "version": "1.1.0",
-    "homepage": "https://yarnspinner.dev",
-    
-    "license": "MIT",
-    "keywords": [
-        "yarn spinner", "yarnspinner", "yarn", "game writing", "narrative", "tree", "branch", "branching", "dialog", "dialogue"
+  "name": "yarn-spinner",
+  "displayName": "Yarn Spinner",
+  "description": "Adds support for the Yarn Spinner dialogue language.",
+  "version": "1.1.0",
+  "homepage": "https://yarnspinner.dev",
+  "license": "MIT",
+  "keywords": [
+    "yarn spinner",
+    "yarnspinner",
+    "yarn",
+    "game writing",
+    "narrative",
+    "tree",
+    "branch",
+    "branching",
+    "dialog",
+    "dialogue"
+  ],
+  "qna": false,
+  "publisher": "SecretLab",
+  "icon": "icon.png",
+  "galleryBanner": {
+    "color": "#C5E29B",
+    "theme": "light"
+  },
+  "engines": {
+    "vscode": "^1.44.0"
```

Comment:
This had to be bumped up from `1.38.0` since `1.44.0` was the most recent version to fully support custom editors (this is some cutting edge stuff! :sunglasses: )